go.mod

```diff
@@ -768,12 +798,20 @@ replace k8s.io/cri-api => k8s.io/cri-api v0.25.5
 // Use custom Trivy fork to reduce binary size
 // Pull in replacements needed by upstream Trivy
 replace (
-	github.com/aquasecurity/trivy => github.com/DataDog/trivy v0.0.0-20240118082257-7f12ba39deb4
+	github.com/aquasecurity/trivy => github.com/DataDog/trivy v0.0.0-20240228154827-8bb996511de3
 	github.com/saracen/walker => github.com/DataDog/walker v0.0.0-20230418153152-7f29bb2dc950
-	github.com/spdx/tools-golang => github.com/spdx/tools-golang v0.3.0
-	oras.land/oras-go => oras.land/oras-go v1.1.1
+	// oras 1.2.2 is incompatible with github.com/docker/docker v24.0.2
+	// cf. https://github.com/oras-project/oras-go/pull/527
+	oras.land/oras-go => oras.land/oras-go v1.2.5
 )
 
+// Temporarely use forks of trivy libraries to use lazy initialization of zap loggers.
+// Patch was pushed upstream but maintainers would prefer moving to slog once 1.22 is out
+replace github.com/aquasecurity/trivy-db => github.com/lebauce/trivy-db v0.0.0-20240228172000-42caffdaee3f
```

Comment:
IMO we should not depends on a personal fork here. 
can we fork trivy-db in the datadog github org  too 🙏

Reply:
Done.

Reply:
The change is https://github.com/DataDog/trivy-db/commits/new-lazy-logger/.